lib/Search/MessageSearch.php

```diff
@@ -120,6 +121,13 @@ public function search(IUser $user, ISearchQuery $query): SearchResult {
 				continue;
 			}
 
+			if ($room->getLobbyState() !== Webinary::LOBBY_NONE) {
+				$participant = $room->getParticipant($user->getUID(), false);
```

Comment:
Why is the order of different to the if clause above and split into two parts?

Reply:
because in the `Current` it's a one-time thing, so it does not matter as much, while here we would potentially set up your user in hundreds of conversations even if they don't have a lobby in the end.